### PR TITLE
Issue 6938: sort values for String Set properties 

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6938 

First commit to format xdocs/javadoc.
Second commit to fix `XdocsPagesTest#getModulePropertyExpectedValue`

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java [hunk 1]
```diff
@@ -1063,40 +1065,24 @@ else if (fieldClass == double[].class) {
...
-                        newArray[index] = (String) next;
-                        index++;
-                    }
-
-                    value = newArray;
-                }
-
-                if (value != null && Array.getLength(value) > 0) {
-                    if (Array.get(value, 0) instanceof Number) {
-                        final String[] newArray = new String[Array.getLength(value)];
```
Comment (pbludov) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java]: This code has no effect and removed. Currently none of the checks stores `TOKENS` as a set of strings.

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java [hunk 2]
```diff
@@ -1063,40 +1065,24 @@ else if (fieldClass == double[].class) {
...
-                                    .intValue());
-                        }
-
-                        value = newArray;
-                    }
-
-                    result = Arrays.toString((Object[]) value).replace("[", "")
-                            .replace("]", "");
+                if (value == null) {
+                    result = "";
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java]: This is changing the displayed value if a collection/array of string was ever `null`.
We would display "null" before but now you will be displaying an empty string.

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java [hunk 3]
```diff
@@ -1063,40 +1065,24 @@ else if (fieldClass == double[].class) {
...
+                    else {
+                        final Object[] array = (Object[]) value;
+                        valuesStream = Arrays.stream(array);
+                    }
+                    result = valuesStream
+                        .map(String.class::cast)
+                        .sorted()
+                        .collect(Collectors.joining(", "));
                 }
-
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java]: Please restore empty line. It makes a clearer separation between the 2 blocks.

Comment (pbludov) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java]: done

**Pull request merged**